.github/matrices/builds.json

```diff
@@ -0,0 +1,35 @@
+{
+  "pr": {
+    "include": [
+      {
+        "platform": "iOS",
+        "xcode_version": "16.4",
+        "macos-version": "15"
+      }
+    ]
+  },
+  "main": {
+    "include": [
+      {
+        "platform": "iOS",
+        "xcode_version": "16.4",
```

Comment:
should we also have an entry for `26.0`?

Reply:
in the future. it's very fake for now.